normalized.json

```diff
@@ -187,6 +187,13 @@
     ],
     "type": "object",
     "properties": {
+        "documentType": {
```

Comment:
Yup sure. Way to do that is adding null http://spacetelescope.github.io/understanding-json-schema/reference/generic.html